src/main/java/org/assertj/vavr/internal/Maps.java

```diff
@@ -388,10 +429,14 @@ private static void assertNotNull(AssertionInfo info, Map<?, ?> actual) {
     }
 
     private static <K> Predicate<K> presentIn(Set<K> elements) {
-        return elem -> elements.contains(elem);
+        return elements::contains;
+    }
+
+    private <V> Predicate<V> valuePresentIn(Seq<V> elements) {
+        return elements::contains;
     }
 
-    private static <K> Predicate<K> notPresentIn(Seq<K> elements) {
+    private static <V> Predicate<V> valueNotPresentIn(Seq<V> elements) {
         return elem -> !elements.contains(elem);
```

Comment:
Maybe `!valuePresentIn` - use the method defined above.

Reply:
Because `valuePresentIn` returns `Predicate` not `boolean`

Reply:
You're right. However there's a `negate` method on `Predicate`. Up to you :)